packages/msw/README.md

```diff
@@ -0,0 +1,49 @@
+# `hops-msw`
+
+[![npm](https://img.shields.io/npm/v/hops-msw.svg)](https://www.npmjs.com/package/hops-msw)
+
+**Please see the [main Hops Readme](../../DOCUMENTATION.md) for general information and a Getting Started Guide.**
+
+This is a [preset for Hops](../../DOCUMENTATION.md#presets) that can be used to set up a [Mock Service Worker](https://www.npmjs.com/package/msw) for your Hops app. It allows to mock REST and GraphQL APIs in unit and integration tests.
+
+This package (together with `jest-preset-hops`) takes care of setting up the MSW mock server already, so if you use it for unit-testing, you do not need to do this yourself.
+
+**Example:**
+
+```javascript
+import { render, waitFor } from '@testing-library/react';
+import { mockServer, graphql } from 'hops-msw';
+import { withApolloTestProvider } from 'hops-react-apollo';
+
+it('loads graphql data', async () => {
+  mockServer.use(
+    graphql.query('search', (req, res, ctx) => {
+      return res(
+        ctx.data({
+          __typename: 'Query',
+          user: {
+            __typename: 'User',
+            name: 'Something',
+          },
+        })
+      );
+    })
+  );
+
+  const { getByText } = render(withApolloTestProvider(<HomeWithData />));
+
+  await waitFor(() => getByText('Something'));
+});
+```
+
+Read the [MSW documentation](https://mswjs.io/) for more information on how to write mocks.
+
+### Installation
+
+Add this preset to your existing Hops React project:
+
+```bash
+npm install --save hops-msw
+```
+
+In order to use it for unit-testing, you also need to install the [`jest-preset-hops`](../jest-preset-hops/README.md) package.
```

Comment:
I think we should add a usage description and mention the environment variable here